packages/block-editor/src/hooks/border-color.js

```diff
@@ -83,11 +89,10 @@ export function BorderColorEdit( props ) {
 		<ColorGradientControl
 			label={ __( 'Color' ) }
 			colorValue={ colorValue }
-			colors={ colors }
-			gradients={ undefined }
-			disableCustomColors={ disableCustomColors }
-			disableCustomGradients={ disableCustomGradients }
 			onColorChange={ onChangeColor }
+			clearable={ false }
```

Comment:
I believe this `clearable` prop was only added to https://github.com/WordPress/gutenberg/pull/33743 because the control could be reset via the `ToolsPanel` menu. Given this PR is for before that lands perhaps it should be omitted here. 

It's only a small thing. I don't feel strongly either way.